pkg/iostreams/iostreams.go

```diff
@@ -396,6 +426,10 @@ func System() *IOStreams {
 		io.progressIndicatorEnabled = true
 	}
 
+	if stdoutIsTTY && assumeTrueColor {
+		io.alternateScreenBufferEnabled = true
+	}
```

Comment:
For clarity could we create another variable that is equivalent to `assumeTrueColor` but named something like `enableAlternateScreenBuffer`?

Reply:
All this really represents is virtual terminal processing is supported, which returns conditionally on Windows and returns always true for other platforms in lines 406 through 411. I don't think introducing yet another local variable makes sense, but if the variable name `assumeTrueColor` is misleading, perhaps that should be renamed to something like `supportsVirtualTerminalProcessing` or `supportsVT100` or something like that. Thoughts?